src/onegov/org/models/extensions.py

```diff
@@ -452,12 +461,25 @@ def apply_model(self, model):
         builder = WTFormsClassBuilder(PeoplePageForm)
         builder.set_current_fieldset(fieldset_label)
 
-        for person in self.get_selectable_people(request):
+        selectable_people = self.get_selectable_people(request)
+        if selectable_people:
+            builder.add_field(
+                field_class=BooleanField,
+                field_id='western_ordered',
+                label=_("Use Western ordered names"),
+                description=_("For instance Franz Müller instead of Müller "
+                              "Franz"),
+                required=False,
+                default=self.show_western_name_order,
+            )
+        for person in selectable_people:
             field_id = fieldset_id + '_' + person.id.hex
+            name = ' '.join([person.first_name, person.last_name]) if (
```

Comment:
```suggestion
            name =f '{person.first_name} {person.last_name}' if (
```
Don't use a join with a static sequence, just write the f-string or use `str.format`.